### PR TITLE
fixed conventions of utils and control at least I hope

### PR DESCRIPTION
**Pull request by Yoel-Perman-Brilliant**

### CODE REVIEW

**Comment by Yoel-Perman-Brilliant** [comment]
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: src/main/java/com/spikes2212/control/noise/RunningAverageFilter.java [hunk 1]
```diff
@@ -5,7 +5,6 @@
     private double sum = 0;
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/noise/RunningAverageFilter.java:5]: add `@author`

Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/noise/RunningAverageFilter.java:5]: add documentation for `sum`

File: src/main/java/com/spikes2212/control/noise/RunningAverageFilter.java [hunk 2]
```diff
@@ -5,7 +5,6 @@
...
     private int count = 0;
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/noise/RunningAverageFilter.java:6]: add documentation for `count`

File: src/main/java/com/spikes2212/util/PigeonWrapper.java [hunk 3]
```diff
@@ -7,6 +7,7 @@
     public double[] values = new double[3];
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/PigeonWrapper.java:7]: add `@author`

File: src/main/java/com/spikes2212/util/PigeonWrapper.java [hunk 4]
```diff
@@ -15,41 +16,70 @@ public PigeonWrapper(TalonSRX talonSRX) {
...
     public void reset() {
         setYaw(0);
     }
 
-    public void calibrate() {
-        calibrate(0);
-    }
-
+    /**
+     * calibrates the piegon wrapper based on the yaw sent
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/PigeonWrapper.java:70]: the pigeon in general, not specifically the wrapper

File: src/main/java/com/spikes2212/util/RepeatCommand.java [hunk 5]
```diff
@@ -12,6 +12,11 @@
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/RepeatCommand.java:12]: remove the redundant `@link`
and use `a commands that...`

Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/RepeatCommand.java:12]: add `@author`

File: src/main/java/com/spikes2212/util/XboXUID.java [hunk 6]
```diff
@@ -6,95 +6,52 @@
 
 public class XboXUID extends Joystick {
 
+
+    XboxController xbox;
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/XboXUID.java:10]: add visibility

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: src/main/java/com/spikes2212/control/PIDSettings.java [hunk 7]
```diff
@@ -3,34 +3,44 @@
 import java.util.function.Supplier;
 
 /**
- * The PID constants used in a {@link PIDLoop}, and the error tolerance and loop wait time.
+ * @author Ergoold
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/PIDSettings.java:6]: that's not the author's real name

File: src/main/java/com/spikes2212/control/noise/NoiseFilter.java [hunk 8]
```diff
@@ -1,7 +1,9 @@
 package com.spikes2212.control.noise;
 
+/**
+ * @author Simon Kharmatsky
+ */
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/noise/NoiseFilter.java:5]: add class documentation

File: src/main/java/com/spikes2212/util/PigeonWrapper.java [hunk 9]
```diff
@@ -3,10 +3,14 @@
 import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 import com.ctre.phoenix.sensors.PigeonIMU;
 
+/**
+ * @author Simon Kharmatsky
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/PigeonWrapper.java:7]: wrong author

File: src/main/java/com/spikes2212/util/RepeatCommand.java [hunk 10]
```diff
@@ -5,13 +5,19 @@
 import java.util.Set;
 
 /**
- * This {@link Command} repeats a sequence of commands indefinitely
+ * @author Ergoold
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/RepeatCommand.java:8]: not the author's real name

File: src/main/java/com/spikes2212/util/XboXUID.java [hunk 11]
```diff
@@ -4,97 +4,56 @@
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.button.*;
 
+/**
+ * @author Simon Kharmatsky
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/XboXUID.java:8]: wrong author

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: src/main/java/com/spikes2212/control/FeedForwardController.java [hunk 12]
```diff
@@ -1,7 +1,8 @@
 package com.spikes2212.control;
 
 /**
- * This class represents a FeedForwardController.
+ * @author Simon Kharmatsky
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/FeedForwardController.java:4]: move this below the class' documentation

File: src/main/java/com/spikes2212/control/FeedForwardSettings.java [hunk 13]
```diff
@@ -2,7 +2,15 @@
 
 import java.util.function.Supplier;
 
+/**
+ * @author Simon Kharmatsky
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/FeedForwardSettings.java:6]: move this below the class' documentation

File: src/main/java/com/spikes2212/control/PIDSettings.java [hunk 14]
```diff
@@ -3,34 +3,44 @@
 import java.util.function.Supplier;
 
 /**
- * The PID constants used in a {@link PIDLoop}, and the error tolerance and loop wait time.
+ * @author Eran Goldshtein
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/PIDSettings.java:6]: move this below the class' documentation

File: src/main/java/com/spikes2212/control/noise/NoiseFilter.java [hunk 15]
```diff
@@ -1,7 +1,13 @@
 package com.spikes2212.control.noise;
 
+/**
+ * @author Simon Kharmatsky
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/noise/NoiseFilter.java:4]: move this below the class' documentation

File: src/main/java/com/spikes2212/util/PigeonWrapper.java [hunk 16]
```diff
@@ -3,10 +3,14 @@
 import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 import com.ctre.phoenix.sensors.PigeonIMU;
 
+/**
+ * @author Tal Sitton
+ */
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/PigeonWrapper.java:8]: add class documentation

File: src/main/java/com/spikes2212/util/RepeatCommand.java [hunk 17]
```diff
@@ -5,13 +5,19 @@
 import java.util.Set;
 
 /**
- * This {@link Command} repeats a sequence of commands indefinitely
+ * @author Eran Gold
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/RepeatCommand.java:8]: move this below the class' documentation

File: src/main/java/com/spikes2212/util/XboXUID.java [hunk 18]
```diff
@@ -4,97 +4,56 @@
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.button.*;
 
+/**
+ * @author Tal Sitton
+ */
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/XboXUID.java:9]: add class documentation

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: src/main/java/com/spikes2212/control/PIDSettings.java [hunk 19]
```diff
@@ -3,34 +3,44 @@
 import java.util.function.Supplier;
 
 /**
- * The PID constants used in a {@link PIDLoop}, and the error tolerance and loop wait time.
+ * The PID constants used in a PIDLoop, and the error tolerance and loop wait time.
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/PIDSettings.java:6]: PIDLoop doesn't exist

File: src/main/java/com/spikes2212/control/FeedForwardController.java [hunk 20]
```diff
@@ -1,7 +1,8 @@
 package com.spikes2212.control;
 
 /**
- * This class represents a FeedForwardController.
+ * a class that represents a FeedForwardController
+ * @author Simon Kharmatsky
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/FeedForwardController.java:5]: its Tuval

File: src/main/java/com/spikes2212/util/PigeonWrapper.java [hunk 21]
```diff
@@ -3,10 +3,15 @@
 import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 import com.ctre.phoenix.sensors.PigeonIMU;
 
+/**
+ * a wrapper class for the pigeon sensor
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/PigeonWrapper.java:7]: the `pigeon IMU sensor`

File: src/main/java/com/spikes2212/util/PigeonWrapper.java [hunk 22]
```diff
@@ -15,41 +20,70 @@ public PigeonWrapper(TalonSRX talonSRX) {
         pigeon = new PigeonIMU(talonSRX);
     }
 
+    /**
+     * @return the X axis from they gyro
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/PigeonWrapper.java:24]: they?

File: src/main/java/com/spikes2212/util/PigeonWrapper.java [hunk 23]
```diff
@@ -15,41 +20,70 @@ public PigeonWrapper(TalonSRX talonSRX) {
...
+     */
     public double getX() {
         pigeon.getAccumGyro(values);
         return values[0];
     }
 
+    /**
+     * @return the Y axis from they gyro
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/PigeonWrapper.java:32]: ditto

File: src/main/java/com/spikes2212/util/PigeonWrapper.java [hunk 24]
```diff
@@ -15,41 +20,70 @@ public PigeonWrapper(TalonSRX talonSRX) {
...
+     */
     public double getY() {
         pigeon.getAccumGyro(values);
         return values[1];
     }
 
+    /**
+     * @return the Z axis from they gyro
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/PigeonWrapper.java:40]: ditto

File: src/main/java/com/spikes2212/util/RepeatCommand.java [hunk 25]
```diff
@@ -5,13 +5,19 @@
 import java.util.Set;
 
 /**
- * This {@link Command} repeats a sequence of commands indefinitely
+ * a command that repeats a sequence of commands indefinitely
+ * @author Eran Gold
  */
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/util/RepeatCommand.java:11]: misplaced white-space

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: src/main/java/com/spikes2212/control/PIDSettings.java [hunk 26]
```diff
@@ -3,7 +3,7 @@
 import java.util.function.Supplier;
 
 /**
- * The PID constants used in a PIDLoop, and the error tolerance and loop wait time.
+ * The constants used in a PID calculation, and the error tolerance and loop wait time.
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/control/PIDSettings.java:6]: too many `and`s in one sentence

**Review by tal-sitton — APPROVED** [review]

**Review by tal-sitton — CHANGES_REQUESTED** [review]
.

**Review by tal-sitton — APPROVED** [review]

**Pull request merged**